src/sentry/static/sentry/app/__mocks__/api.tsx

```diff
@@ -119,7 +119,7 @@ class Client {
     if (!response || !mock) {
       // Endpoints need to be mocked
       const err = new Error(
-        `No mocked response found for request:\n\t${options.method || 'GET'} ${url}`
+        `No mocked response found for request: ${options.method || 'GET'} ${url}`
```

Comment:
The newline + tab would push the method and URL into the next line and not print properly.